src/main/java/com/minecolonies/coremod/permissions/ColonyPermissionEventHandler.java

```diff
@@ -97,12 +106,18 @@ private boolean checkBlockEventDenied(final World worldIn, final BlockPos posIn,
         return false;
     }
 
-    private static void cancelEvent(final Event event)
+    private static void cancelEvent(final Event event, @Nullable final EntityPlayer player)
```

Comment:
No we can't because every event gets the player differently.
So we can't actually get the player from the plain event.